packages/gravity-ui-web/src/sass/05-components/03-organisms/03-articles/_00-related-items.scss

```diff
@@ -0,0 +1,17 @@
+.grav-c-related-items {
+  max-width: 540px;
+  margin: 0 auto;
```

Comment:
As per Gravity's layout conventions, components shouldn't set any (max-)width or outer margins on the component itself, so we should remove these lines.

Reply:
If we're going with the approach of turning this into a list, you can add equivalent styles to the `<li>`s within.

Please try to use `rem` rather than `px` though.

Reply:
@james-nash Without this the text block gets extremely long and hard to read. Are you sure we want to remove the max width?

![Screen Shot 2019-05-17 at 2 09 57 PM](https://user-images.githubusercontent.com/643307/57953528-85bde200-78ad-11e9-99c2-f1638552ee34.png)